templates/index.html

```diff
@@ -3,9 +3,37 @@
 <head>
     {% include "includes/header.html" %}
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>jacob.covs.tld</title> <!-- TEMP:REPLACE -->
+
+    <div class="container">
```

Comment:
Pull out navbar into `templates/includes/navbar.html`, which we can then do `{% include "includes/navbar.html" %}` within each page.

Reply:
I was going to, but that assumes that the navbar will be the same on each page, which it won't be:
The current page's highlighting needs changing, and which options are available need to be changed.
Would this still work anyway (i.e. can we dynamically change things in the included navbar.html ?) ??